CMakeLists.txt

```diff
@@ -33,8 +33,16 @@ include(FetchContent)
 include(FindPkgConfig)
 include(AvifExternalProjectUtils)
 
+option(AVIF_ENABLE_NODISCARD "Add [[nodiscard]] to some functions. CMake must be at least 3.21 to request C23." OFF)
```

Comment:
This is copied from line 55 in the original code, with "force C23" changed to "request C23."